cmake/modules/SIMDExt.cmake

```diff
@@ -31,18 +31,22 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
   if(HAVE_ARMV8_CRC)
     message(STATUS " aarch64 crc extensions supported")
   endif()
+  CHECK_C_COMPILER_FLAG(-march=armv8-a+crc+crypto HAVE_ARMV8_CRC_CRYPTO_INTRINSICS)
+  if(HAVE_ARMV8_CRC_CRYPTO_INTRINSICS)
+    set(ARMV8_CRC_COMPILE_FLAGS "${ARMV8_CRC_COMPILE_FLAGS} -march=armv8-a+crc+crypto")
```

Comment:
I think we're misunderstanding each other. The problem is that our compiler does not support -march= or -mcpu= on anything but armv8-a.

However, our assembler handles those additional instructions just fine. So all we need to change is the check from the one he's implementing to something that resembles the block above.